### DISCUSSION TITLE
MDX file are not generated automaticaaly when I run dev server

### DISCUSSION THREAD

**Discussion by irorochad** [fuma-nama/fumadocs#859] [Q&A]
Hey guys,

Greetings, and Thanks for the hard work.

I'm trying to integrate `fumadocs` into my existing frontend project. I want to have something like `website.com/docs` instead of `docs.website.com`.

When I tried creating the project from scratch and creating it's own folder, everything seems to work fine. Now though, I am manually installing it on my existing project. What I did was to basically install the dependencies and manually copied the files and folder into my project.

After doing that, I made some twisting to the `pageLayout`, some other files, and even updated the `.source` folder at the root of the file to get the correct path of the existing the `mdx` file.

Everything seems to be working well, but if I create a new `.mdx` file in the `content` folder, the server doesn't see it (404). 

Also, when I kill the server, the `.source/index.js` file cleans up
`import { toRuntime } from "fumadocs-mdx"
export const docs = []
export const meta = []
`
I don't know what I'm not doing right, nor do I know the codes to share with you. please let me know :)

**Comment by fuma-nama** [comment]
Hmm do you mean creating a file in dev mode doesn't result in an instant update?

**Reply by fuma-nama** [reply]
I'll need a reproduction for this one (via GitHub or CodeSandbox), welcome to open a bug report

**Comment by irorochad** [comment]
Thanks.

I'll keep you posted.  Please also let me know when you do find something :)
